src/main/java/com/skynest/clients/SkyNestBackendClient.java

```diff
@@ -39,41 +38,40 @@ public Response getAllUsers() {
                 .get(ApiConstants.GET_ALL_USERS_PATH);
     }
 
-    public Response getUserById() throws IOException {
-        Response response = getLoggedUser();
-        LoggedUserResponse loggedUserResponse = JsonTransformer.mapResponse(response, LoggedUserResponse.class);
+    public Response getUserById(UUID uuid) {
         return requestMaker()
-                .get(ApiConstants.USERS_PATH + loggedUserResponse.getUuid());
+                .pathParam("uuid", uuid)
```

Comment:
Consider storing the "uuid" string into a variable and use the variable instead. This would reduce the lines to change in this class to 1, should the url path ever change.

Reply:
Right now we are faced with url path changes, so I will definitely store it into a variable.